packages/react-components/src/theming/FluentThemeProvider.tsx

```diff
@@ -34,11 +34,21 @@ export const ThemeContext = createContext<Theme>(defaultTheme);
 
 const initialFluentNorthstarTheme = mergeNorthstarThemes(teamsTheme, {
   componentVariables: {
-    // suppressing teams theme for chat message links to get better styling from Fluent UI Link
+    // suppressing chat message links style from teamsTheme to get better styling from Fluent UI Link
     ChatMessage: {
       linkColor: undefined,
       linkColorMine: undefined
     }
+  },
+  fontFaces: [], // suppressing font faces from teamsTheme
```

Comment:
Please add a detailed comment in the code here describing why this is necessary, and link to the bug / bugs.

This is a workaround for a really pesky problem, so let's make it loud and clear in the code :)

Reply:
Also, for my edification, can you describe why this doesn't break the font/styling for the NorthStar components themselves? Are all the body* styling just spurious (i.e., NorthStar components themselves don't even rely on those?)